src/client/graphics/layers/UnitLayer.ts

```
@@ -279,7 +279,9 @@
       case UnitType.TradeShip:
         this.handleTradeShipEvent(unit);
         break;
+      case UnitType.CargoPlane:
+        this.handleCargoPlaneEvent(unit);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)